statistics/handle/handle.go

```diff
@@ -579,6 +583,103 @@ func (h *Handle) mergePartitionStats2GlobalStats(sc sessionctx.Context,
 	return
 }
 
+func (h *Handle) mergeGlobalStatsTopN(sc sessionctx.Context, wrapper *statistics.StatsWrapper,
```

Comment:
Should we handle the `version` at first? For example, if `version == 1` just return.

Reply:
Will statistics version affect this?

Reply:
I don't know... Pls add some test cases for version1.

Reply:
Only the version2 has the TopN. Maybe for the Version1, we don't need to merge topN?